### PR TITLE
nicholascassar/class-1/insert-licence

### PR DESCRIPTION
**Pull request by nscassar**
Added a LICENCE file.

### CODE REVIEW

**Comment by Right2Drive** [comment]
@nscassar There is nobody assigned to this PR

File: LICENCE [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/*******************************************************
+ * Copyright (C) 2017-2018 James Thorogood <jamesthorogood@ewb.ca>
+ *
+ * This file is part of AgriBoard.
+ * All files in this directory and all subdirectories are also part of AgriBoard.
+ *
+ * AgriBoard and its enclosing files can not be copied and/or distributed without
```
Comment (enochtsang) [inline LICENCE]: cannot is one word

File: README.adoc [hunk 2]
```diff
@@ -0,0 +1,114 @@
+= Agriboard
+
+Agriboard is an administrative dashboard for the agriculture software stack.
```
Comment (enochtsang) [inline README.adoc]: How did this get added. Where did my commits go to add this before lol. @Right2Drive

File: LICENCE (hunk 1 above)
Comment (nscassar) [reply LICENCE]: According to this link, both uses are acceptable.
https://en.oxforddictionaries.com/usage/cannot-or-can-not

File: README.adoc (hunk 2 above)
Comment (nscassar) [reply README.adoc]: Umm would this by any chance be tied to the fact that when I first tried to do the pull request, I had the ‘source’ and ‘destination’ branches backwards? I closed it, then re-made the pull request, this time with the branches in the correct order.

**Comment by enochtsang** [comment]
Create a new PR off of master, I dont' think this is off of master.

**Pull request closed without merging**